client/app/index.html

```diff
@@ -26,6 +26,23 @@
   <body ng-class="bodyClass">
     <section>
       <app-view></app-view>
+      <div class="loading-indicator">
+        <div id="css-logo">
+          <div id="circle">
+            <div></div>
+          </div>
+          <div id="point">
+            <div></div>
+          </div>
+          <div id="bars">
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+          </div>
+        </div>
+        <shadow></shadow>
```

Comment:
Not sure if it was intentional: `<shadow>` tag is an obsolete tag for shadow DOM. Probably here it should be just `<div>`. Also, it's not recommended to use single-word names for _custom_ components (because such names may be used for new HTML tags in future) - custom components should have names with few words separated by `-`